deploy/templates/service-template.yml

```diff
@@ -73,6 +73,9 @@ parameters:
     description: Define the backend authentication mechanism
   - name: MIGRATION_PLANNER_JWK_URL
     description: URL of the x.509 certificate chain that was used to verify the digital signature of the JWT
+  - name: MIGRATION_PLANNER_MIGRATIONS_FOLDER
```

Comment:
Its not used anywhere
I guess its also should be added in the Deployment environment variables declaration